extensions/objects/templates/skill-selector-editor.component.ts

```diff
@@ -36,6 +36,8 @@ export class SkillSelectorEditorComponent implements OnInit, OnDestroy {
   @Input() modalId!: symbol;
   @Input() value!: string;
   @Output() valueChanged = new EventEmitter();
+  initialEntityId: string = '';
```

Comment:
This and the next value should never be set to the empty string, I think.

The empty string indicates an actual entity ID that is the empty string, and similarly for the entity type. If you want to indicate "no value", then use null. Otherwise maybe don't set it at all (similar to the others). 

Would that be possible? (Still need to verify that it works fine in that case, but feel free to merge after making the changes and testing and verifying that they work.)

Reply:
Thanks @seanlip, I made the changes and yes it works as expected.